### ISSUE TITLE
Error in zigbee websocket service if device has no room associated

### ISSUE THREAD

**Issue by LucidityCrash** [asantaga/wiserHomeAssistantPlatform#275]
Card is blank and I see the following error whenever a page with the card is loaded : 
```
2022-07-11 17:27:15 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [139783479717376] Error handling message: Unknown error (unknown_error)
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 27, in _handle_async_response
await func(hass, connection, msg)
File "/config/custom_components/wiser/websockets.py", line 513, in websocket_get_zigbee_data
"label": f"{device.name}\n({d.wiserhub.rooms.get_by_device_id(device.id).name})",
AttributeError: 'NoneType' object has no attribute 'name'
```

Hub firmware is listed as 3.10.5
Trv firmare is listed as 0000dac0
I can see the zigbee connection details : 
![image](https://user-images.githubusercontent.com/49124523/178332030-dccd6442-687a-40cc-aaf6-4ce679175281.png)

**Comment by msp1974** [comment]
You have a device that doesnt belong to a room and i forgot to accomodate that (had similar issue before).  Will fix in next update but for now add everything toma room and it should work.

**Comment by LucidityCrash** [comment]
Sweet ... thanks for that ... I have a single plug as a repeater that I've not put into a room ... done that and map is now working

**Comment by asantaga** [comment]
if it makes you feel better I had precisely the same :-)

**Comment by msp1974** [comment]
This should now be fixed in v3.1.2
